shared/domains/gi/db.js

```diff
@@ -37,9 +37,15 @@ export default (sbp('sbp/selectors/register', {
   'gi.db/set': function (key: string, value: string): Promise<void> {
     return Promise.resolve(sbp('okTurtles.data/set', key, value))
   },
+  'gi.db/delete': function (key: string) {
+    return Promise.resolve(sbp('okTurtles.data/delete', key))
+  },
   'gi.db/log/logHEAD': function (contractID: string): string {
     return `${contractID}-HEAD`
   },
+  'gi.db/log/isLogHEAD': function (key: string): string | boolean {
+    return key.slice(-5) === '-HEAD' ? key.slice(0, -5) : false
+  },
```

Comment:
What do you say about changing the name of the selector to `getLogHEAD` or such? I had this same thought while writing it, but I think also that perhaps two selector calls are a bit excessive for the required use (as an aside, I'm not sure I like the current implementation that changes the key name for contract heads like, i.e., `gi.db/log/logHEAD`, although I don't have a proposal for that at the moment)

Reply:
It seems to be difficult to come up with a better name for this one. While `contractIdFromLogHEAD` is a bit long, it's easy to understand.
- Afaik, `get-` and `set-` prefixed names like `getLogHEAD` are not common in the codebase. I'm not saying these names are inherently bad, but maybe avoid them here for consistency.
- `key.endsWith('-HEAD') ?` could be used instead of `key.slice(-5) === '-HEAD' ?`. There should be no compatibility problem with older browsers thanks to build tools, therefore we can use newer JS functions.
- However, I think the name must be changed to `contractIdFromLogHEAD` if you wish to keep it valid camelCase. IIRC, in camel case acronyms may be written all-caps at the end of a name, but not in the middle.

Reply:
Personnaly I would have used `key.slice(0, -'HEAD'.length)` without defining a `headSuffix` constant. But that's no big deal. Do you have any thought on this @greg ?